src/compass/utils/validate_cslc.py

```diff
@@ -110,12 +110,33 @@ def compare_cslc_products(file_ref, file_sec):
         return
 
     # Compare amplitude of reference and generated CSLC products
-    print('Check mean real part difference between CSLC products is < 1.0e-5')
-    assert np.allclose(slc_ref.real, slc_sec.real,
-                       atol=0.0, rtol=1.0e-5, equal_nan=True)
-    print('Check mean imaginary part difference between CSLC products is < 1.0e-5')
-    assert np.allclose(slc_ref.imag, slc_sec.imag,
-                       atol=0.0, rtol=1.0e-5, equal_nan=True)
+    np.seterr(invalid='ignore')
+    diff_real = np.abs((slc_ref.real - slc_sec.real) / slc_sec.real)
+    diff_imag =  np.abs((slc_ref.imag - slc_sec.imag) / slc_sec.imag)
```

Comment:
`np.abs` redundant here no? The `real` and `imag` properties should not leave any imaginary numbers.

Reply:
We want to compute the absolute difference between real and/or imaginary parts and we don't care about the sign. In this case, `np.abs` is not redundant. Correct me if I am wrong @seongsujeong

Reply:
Thank you for the explanation. I failed to understand that the absolute difference was being computed.